src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/caseload/CaCaseloadIntegrationTest.kt

```diff
@@ -162,6 +166,83 @@ class CaCaseloadIntegrationTest : IntegrationTestBase() {
     }
   }
 
+  @Nested
+  inner class CaseAdminSearch {
+    val request = PrisonUserSearchRequest(
+      query = "Person",
+      prisonCaseload = "BAI",
+    )
+
+    @Test
+    fun `Get forbidden (403) when incorrect roles are supplied`() {
+      val result = webTestClient.post()
+        .uri(SEARCH_PRISONERS_CA_CASELOAD)
+        .bodyValue(request)
+        .accept(APPLICATION_JSON)
+        .headers(setAuthorisation(roles = listOf("ROLE_CVL_WRONG ROLE")))
+        .exchange()
+        .expectStatus().isForbidden
+        .expectStatus().isEqualTo(FORBIDDEN.value())
+        .expectBody(ErrorResponse::class.java)
+        .returnResult().responseBody
+
+      assertThat(result?.userMessage).contains("Access Denied")
+    }
+
+    @Test
+    fun `Unauthorized (401) when no token is supplied`() {
+      webTestClient.post()
+        .uri(SEARCH_PRISONERS_CA_CASELOAD)
+        .bodyValue(request)
+        .accept(APPLICATION_JSON)
+        .exchange()
+        .expectStatus().isEqualTo(UNAUTHORIZED.value())
+    }
+
+    @Test
+    @Sql(
+      "classpath:test_data/seed-ca-caseload-licences.sql",
+    )
+    fun success() {
```

Comment:
I prefer a little more than just success for the method test name,... I'm evaluating my approach now. Is this a common approach you use?

Reply:
On second thoughts it's probably better to give a little more as you could have many success full out comes in a test of an out put...